packages/auth/src/grant/routes.ts

```diff
@@ -144,12 +154,22 @@ async function createPendingGrant(
   const { body } = ctx.request
   const { grantService, interactionService, config } = deps
   if (!body.interact) {
-    ctx.throw(400, 'interaction_required', { error: 'interaction_required' })
+    ctx.throw(400, 'interaction_required', {
+      error: {
+        code: 'interaction_required',
+        description: 'missing required request field'
```

Comment:
Should this be different from the `invalid_client` description? Maybe 
```
missing required request field 'interact'
```
and 
```
missing required request field 'client'
```
respectively.